### PR TITLE
SPEC file: sync package name with downstream

### PR DESCRIPTION
**Pull request by clebergnu**
This is an incremental work to reduce the differences between downstream and upstream packages.

For now, the package is being renamed to be the same accross upstream and downstream.

### CODE REVIEW

**Comment by clebergnu** [comment]
[Backport Official Fedora SPEC files](https://trello.com/c/uWKfxZHu/887-backport-official-fedora-spec-files)

File: selftests/functional/test_basic.py [hunk 1]
```diff
@@ -114,9 +114,10 @@ def probe_binary(binary):
 # but their behavior and flags are in most cases different.
 GNU_ECHO_BINARY = probe_binary('echo')
 if GNU_ECHO_BINARY is not None:
-    echo_manpage = process.run('man %s' % os.path.basename(GNU_ECHO_BINARY)).stdout
-    if '-e' not in echo_manpage:
-        GNU_ECHO_BINARY = probe_binary('gecho')
+    if probe_binary('man') is not None:
+        echo_manpage = process.run('man %s' % os.path.basename(GNU_ECHO_BINARY)).stdout
+        if '-e' not in echo_manpage:
+            GNU_ECHO_BINARY = probe_binary('gecho')
```
Comment (lmr) [inline selftests/functional/test_basic.py]: Looks good. The commit message has a typo, though - `availabled`

Comment (clebergnu) [inline selftests/functional/test_basic.py]: Good catch.  (Force) pushed a fix to the commit message.

**Comment by apahim** [comment]
We have to track those changes in our docs. I created a card for it (https://trello.com/c/kwRyFgZc/959-docs-update-installing-avocado).

**Pull request merged**